setup generic converter for vavr.Value

julianps opened the pull request:
Adds a Modelmapper-Module for Converting io.vavr.Value

Commit:
Work in progress

Commit:
Added AssertJ and Tests

Commit:
changed Readme

chhsiao90 commented on pom.xml:
```diff
@@ -32,6 +32,11 @@
+			<groupId>org.assertj</groupId>
+			<artifactId>assertj-core</artifactId>
+			<version>3.9.1</version>
+		</dependency>
```
the scope should be test

chhsiao90 commented:
The pull request looks good to me, thanks for creating the module! I will link this module in modelmapper's readme and website.

Commit:
- Changed group/path to com.github.julianps
- added surefire for automatic test execution
- added scope test for assertj
- fixed warnings in pom and Stubfactory

julianps commented:
Thank you very much for the review @chhsiao90 !

Commit:
- Changed name of the artifact to be more precise in the naming
- added .travis.yml

Commit:
Updated ReadMe.md

Commit:
added infotext

Commit:
[maven-release-plugin] prepare for next development iteration

Commit:
[maven-release-plugin] prepare release modelmapper-vavr-module-0.9.0

Commit:
introduced release and deploy management

Commit:
changed gitignore

Commit:
Delete pom.properties

Commit:
Delete createdFiles.lst

Commit:
Delete com.github.julianps.modelmapper.ValueConverterTest.txt

Commit:
Delete TEST-com.github.julianps.modelmapper.ValueConverterTest.xml

Commit:
Delete inputFiles.lst

Commit:
Delete createdFiles.lst

Commit:
Delete inputFiles.lst

Commit:
Changed Readme
